docs/pbc/all/ratings-reviews/202212.0/third-party-integrations/integrate-bazaarvoice.md

```diff
@@ -50,10 +51,17 @@ You can do the administration work on the Bazaarvoice reviews from the [Bazaarvo
 
 ### 2. Add Bazaarvoice domain to your allowlist
 
-To enable your customers to leave reviews on your products, you must add the Bazaarvoice domain to your **Content Security Policy** allowlist. To do that, change your `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
+To enable your customers to leave reviews on your products, you must add the Bazaarvoice domain to your **Content Security Policy** allowlist. 
 
-In the `deploy.yml` file, introduce the required changes: 
 
+To do that, change your `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
```

Comment:
@danielsantos-spryker can you please elaborate on "if changing the environment variable is not possible" - what do you mean and how can this be done?

Reply:
TBH I don't know why it could not be possible. Let's remove this sentence.